`PerformanceTestSuite` Fails to Run

* JGraphT version: f1c32a5
 * Java version (java -version)/platform: OpenJDK 17.0.9 LTS / Windows 11 23H2 (build 22631.2861)

### Description

> ```mvn -Dtest=PerformanceTestSuite test```
> 
> This one is failing [...]  (It turns out even on master it was already failing due to the unnamed module issue, [...])
> 
> _Originally posted by @jsichi in https://github.com/jgrapht/jgrapht/issues/1163#issuecomment-1869872895_

`PerformanceTestSuite` fails to run due to [`java.lang.reflect.InaccessibleObjectException`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/reflect/InaccessibleObjectException.html).

### Steps to reproduce (small coding example)

```bash
mvn -Dtest=PerformanceTestSuite test
```

### Expected behaviour

Performance tests run successfully.

### Actual Behavior

```text
[ERROR] Errors: 
[ERROR]   MaximalCliqueEnumerationPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.clique.MaximalCliqueEnumerationPerformanceTest.testMaximalCliqueRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.clique" to unnamed module @197d671
[ERROR]   MaximumFlowAlgorithmPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.flow.MaximumFlowAlgorithmPerformanceTest.testRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.flow" to unnamed module @197d671
[ERROR]   DirectedAcyclicGraphPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.graph.DirectedAcyclicGraphPerformanceTest.testDirectedAcyclicGraphRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.graph" to unnamed module @197d671
[ERROR]   DirectedAcyclicGraphVSStaticGraphPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.graph.DirectedAcyclicGraphVSStaticGraphPerformanceTest.testDirectedAcyclicGraphVSStaticGraphRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.graph" to unnamed module @197d671
[ERROR]   GraphPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.graph.GraphPerformanceTest.testRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.graph" to unnamed module @197d671
[ERROR]   LowestCommonAncestorAlgorithmPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.lca.LowestCommonAncestorAlgorithmPerformanceTest.testRandomForestBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.lca" to unnamed module @197d671
[ERROR]   MaximumCardinalityBipartiteMatchingPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.matching.MaximumCardinalityBipartiteMatchingPerformanceTest.testRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.matching" to unnamed module @197d671
[ERROR]   PathGrowingWeightedMatchingPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.matching.PathGrowingWeightedMatchingPerformanceTest.testPathGrowingRandomGraphBenchmark() throws org.openjdk.jmh.runner.RunnerException accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.matching" to unnamed module @197d671
[ERROR]   DijkstraShortestPathPerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.shortestpath.DijkstraShortestPathPerformanceTest.testBenchmark() accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.shortestpath" to unnamed module @197d671
[ERROR]   MinimumSpanningTreePerformanceTest.initializationError ? InaccessibleObject Unable to make public void org.jgrapht.perf.spanning.MinimumSpanningTreePerformanceTest.testBenchmarkDenseGraphs() accessible: module org.jgrapht.core does not "exports org.jgrapht.perf.spanning" to unnamed module @197d671
[INFO]
[ERROR] Tests run: 10, Failures: 0, Errors: 10, Skipped: 0
```

--- a/jgrapht-core/pom.xml
+++ b/jgrapht-core/pom.xml
@@ -81,7 +81,21 @@
                     <execution>
                         <id>default-test</id>
                         <configuration>
-                            <argLine>--add-exports org.jgrapht.core/org.jgrapht.interfaces=ALL-UNNAMED</argLine>
+                            <argLine>
+                                --add-exports org.jgrapht.core/org.jgrapht.interfaces=ALL-UNNAMED
+                                <!-- 
+                                    Manually export performance test packages
+                                    Ref: https://github.com/jgrapht/jgrapht/issues/1170
+                                -->
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.clique=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.connectivity=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.flow=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.graph=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.lca=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.matching=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.shortestpath=ALL-UNNAMED
+                                --add-exports org.jgrapht.core/org.jgrapht.perf.spanning=ALL-UNNAMED
+                            </argLine>
                             <excludes>
                                 <exclude>**/perf/**</exclude>
                             </excludes>

